modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -1,43 +1,201 @@
 
-Anypoint Object Store v2 (Object Store v2) provides a REST API that enables you to get and store key-value pairs.
+The Object Store v2 APIs enable you to use REST to temporarily store and retrieve key-value pairs in a single
+application and to retrieve Object Store usage statistics for your organization.
 
-An object store works with a single application to temporarily store application data.
-To authenticate access, you need the client ID and client secret for the application, which Object Store
-displays in the Runtime Manager in Anypoint Platform.
-See <<View the App Client ID and Secret>>.
+Object Store provides these APIs:
 
-The maximum number of keys you can obtain in each GET operation is 25.
+* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2/[Object Store API]
+* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2-stats/[Object Store Stats API]
 
 
-== Object Store v2 Operations
+== Authenticate with the Object Store APIs 
 
-The Object Store connector reads and writes in the `key,value` format; whereas Object Store v2 API reads and writes `key,list` values.
+* Object Store API
++
+To authenticate with the Object Store API, use the client ID and client secret for the app to get a bearer token.
++
+See <<Get the Bearer Token for the App>>.
+* Object Store Stats API
++
+To authenticate with the Object Store Stats API, use the Anypoint username and password to get the access token.
++
+See <<Get the Access Token for the User>>.
+
+
+
+== Prerequisites 
+
+* Create an app that uses Object Store v2 and deploy it to CloudHub.
++
+See xref:runtime-manager::deploying-to-cloudhub.adoc[Deploy to CloudHub].
+* Store a key-value pair in the object store. 
++
+See xref:osv2-guide.adoc#send-data-to-an-object-store[Send Data to an Object Store].
+* <<Get the Access Token for the Anypoint Platform User,Get the access token for the Anypoint Platform user>>.
+* <<Get the Organization ID,Get the organization ID>>.
+* <<Get the Environment ID,Get the environment ID>>.
+* <<Get the Bearer Token for the App,Get the bearer token>>.
+
+=== Get the Access Token for the Anypoint Platform User 
+
+To access the Object Store Stats API, you must first authenticate with the Anypoint Access Management API by getting the access token (ACCESS_TOKEN) for the Anypoint Platform user.
+You also need this token to access the Anypoint Platform API to retrieve your environment and organization IDs.
+
+The default time to live (TTL) for the access token is 60 minutes.
+
+To get the access token, use this `curl` command, replacing USERNAME and PASSWORD with the username and password of the Anypoint Platform user:
+
+[source,console,linenums]
+----
+curl -X POST \
+  -H "Content-Type: application/json" \
+  -d '{"username":"USERNAME", "password":"PASSWORD"}' \
+  https://anypoint.mulesoft.com/accounts/login
```

Comment:
The endpoint to request the access token is x-www-form-urlencoded. application/json is not valid here.
This is the way to curl for requesting the access token.

`curl --location --request POST 'https://anypoint.mulesoft.com/accounts/login' \
--header 'Content-Type: application/x-www-form-urlencoded' \
--data-urlencode 'username=<username>' \
--data-urlencode 'password=<password>'`

Reply:
Interestingly, it works for me! Also described here: https://help.mulesoft.com/s/article/How-to-generate-your-Authorization-Bearer-token-for-Anypoint-Platform